aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -89,25 +87,8 @@ module aptos_framework::reconfiguration {
         !exists<DisableReconfiguration>(@aptos_framework)
     }
 
-    /// Force an epoch change.
-    public entry fun force_reconfigure(account: &signer) acquires Configuration {
```

Comment:
This is only used by the previous write set tooling. But that tooling can just call reconfigure directly.